### PR TITLE
Morga2cd code review 2

### PR DESCRIPTION
**Pull request by CM2inc**
Configured data classes according to kotlin best practices

### CODE REVIEW

**Comment by CM2inc** [comment]
Part 3)
Analysis of the program.
I believe the program to work for it intended purpose although not yet complete 
Was the program available in UC Github on time?
Yes
Is the program documented/commented well enough for you to understand?
Yes
Does the program compile?
Yes
Rationale behind your changes.
The changes I made were to the data classes because I realized the format throughout was inconsistent and not inline with Kotlin best practices
https://kotlinlang.org/docs/data-classes.html
https://riptutorial.com/kotlin/example/9942/creating-dtos--pojos-pocos- 
Links to three specific commits that you made to your own group's GitHub repository after Sprint 1's deadline, and before Sprint 2's deadline.  
[Created a back button in main activity](https://github.com/trey368s/Mobile-Device-Programming/commit/846bb0316d047fc9344a60fc8215664583208639)
[Created a fileprovider to store an image](https://github.com/trey368s/Mobile-Device-Programming/commit/7c5fe2f1bef205668ae8b7da4c599e22e1174df9)
[created a checkout button in MainActivity.kt](https://github.com/trey368s/Mobile-Device-Programming/commit/2ad11ac9cfeace9e9ee1b39ebfef1c47fba2023f)
Learned how to create button in the main view model 
Learned more about kotlin best practices for dtos when code reviewing
Learned to create a file provider to store an image

File: app/src/main/java/com/example/mylittlelibrary/data/Game.kt [hunk 1]
```diff
@@ -1,9 +1,11 @@
 package com.example.mylittlelibrary.data
 
+//import androidx.room.ColumnInfo
+//import androidx.room.PrimaryKey
```
Comment (discospiff) [inline app/src/main/java/com/example/mylittlelibrary/data/Game.kt:4]: Avoid pushing commented code to Version Control.  Version control has tools that makes this unnecessary.
- You can look at history to see what used to be there.
- You can use branches for experimental features.

**Comment by discospiff** [comment]
From this review:
- Commented out code does not belong in version control.
- Use the IDE to automatically format source.  It can do this consistently for all developers.  No need to create a branch and do it manually.
- toStrings(): May be a good idea, if a String representation of these objects are needed.  But then again, Kotlin gives you one by default.
Bottom line... I lean against merging this branch.

**Comment by andrewglanzzz** [comment]
Closing this pull request. Has some good suggestions that may be harvested, but I don't think these are necessary to the project as some things would need to be repurposed for the current state of the project.

**Pull request closed without merging**